### DISCUSSION TITLE
Improve performance for clients

### DISCUSSION THREAD

**Discussion by x3nomorpheus** [ltsp/ltsp#811] [Q&A] [answered]
Hi. Recently I have faced with performance problem in ltsp clients. It related to first launch apps. As I read previous discussions it is not only my problem and in ltsp wiki guides to use image_to_ram for improving. But I would to ask is there any option to improve performance at first launch in others ways?

**Comment by alkisg** [comment]
Start by doing a LAN benchmark, and tell us the results:
https://epoptes.org/documentation/lan-benchmark/

Also mention how long it takes to launch libreoffice for the first time on the server and on a client.

**Comment by x3nomorpheus** [comment]
With all my regrets, I can't test first launch of libreoffice on the server because of using chroot image. Also It hasn't X window system. 
But I setup the server in the same network and reproduce environment for epoptes lan benchmark and here is the results.
![Screenshot_2023-03-24_11-55-30](https://user-images.githubusercontent.com/110904830/227491887-fb243086-ab16-4b4d-9ef6-880a8225ee59.png)

**Comment by alkisg** [comment, marked as answer]
You need to upgrade your network to gigabit to use LTSP efficiently. Currently it's 100 Mbps.

**Reply by brcisna** [reply]
@alkisg 
Thanks for pointing out LAN speed to this gentleman,

Have to tell a story. We started using LTSP, the very first version that came out. At the time,there was no such thing as GBE lan, It was very cool to be booting old dumpster bound giant desktops,. For a long time,,we just kept existing switches in place,,,,most in the ceiling of the school building. Once we went to GBE LAN,,, was like were woking in a  rocket ship. Didn't know any different than 100 mbps for lan for so long,
